lib/src/tree.dart

```diff
@@ -676,7 +679,7 @@ class MixinDefinition extends Directive {
 
 /** Support a Sass @mixin. See http://sass-lang.com for description. */
 class MixinRulesetDirective extends MixinDefinition {
-  final List<RuleSet> rulesets;
+  final List rulesets;
```

Comment:
yeah, exactly, it's not just rulesets.
I haven't (yet) figured out what it can contain...